webapp/portlet/src/main/webapp/vue-apps/common/components/RichEditor.vue

```diff
@@ -587,7 +587,19 @@ export default {
         content = content.replace(/<oembed>(.*)<\/oembed>/g, '');
         content = content.replace(/<oembed>(.*)<\/oembed>/g, `<oembed>${oembedUrl}</oembed>`);
       }
+      if (content.includes('<a') && content.includes('</a>')) {
+        const tempdiv = document.createElement('div');
+        tempdiv.innerHTML = content;
+        const links = tempdiv.getElementsByTagName('a');
+        for (const link of links) {
+          if (!link.href.includes(eXo.env.portal.context) && !link.hasAttribute('target')) {
```

Comment:
You may need to use the same attributes and `origin` comparaison as https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/WEB-INF/gatein-resources.xml#L2439-L2445

Reply:
done